src/Microsoft.DocAsCode.MarkdigEngine.Extensions/TripleColon/ImageExtension.cs

```diff
@@ -104,13 +106,15 @@ public bool TryProcessAttributes(IDictionary<string, string> attributes, out Htm
                 var currentLightbox = string.Empty;
                 var currentBorderStr = string.Empty;
                 var currentBorder = true;
+                var currentLink = string.Empty;
                 if(!obj.Attributes.TryGetValue("type", out currentType))
                 {
                     currentType = "content";
                 }
                 obj.Attributes.TryGetValue("lightbox", out currentLightbox); //it's okay if this is null
                 obj.Attributes.TryGetValue("border", out currentBorderStr); //it's okay if this is null
-                if(!bool.TryParse(currentBorderStr, out currentBorder))
+                obj.Attributes.TryGetValue("link", out currentLink); //it's okay if this is null
```

Comment:
Resolve the link against `_context.GetLink`? In general, all links go through the `GetLink` method, be it external or internal. This is our interface to collect all links for reporting, broken link validation, etc.

Reply:
Thank you!

Reply:
Updated.